### DISCUSSION TITLE
How to Get Dependency Required frameworks and Licensing files for Offline distribution of LOB apps with Winget?

### DISCUSSION THREAD

**Discussion by techyrajeev** [microsoft/winget-cli#3018] [General]
How can I download all "Required frameworks" and "License" files for offline distribution, which was previously possible with Microsoft Store for Business.

![image](https://user-images.githubusercontent.com/4118032/221630155-5042961a-ea95-429f-99ad-b3e607488851.png)

![image](https://user-images.githubusercontent.com/4118032/221629649-f5118eb7-182b-4b7b-b490-48031a6626cb.png)

**Comment by denelon** [comment]
Keep your eye on:
* https://github.com/microsoft/winget-cli/issues/658

**Comment by techyrajeev** [comment]
Hi @denelon 
I have couple of questions -

- can winget support configMgr (SCCM) ?
- can winget be used for org wide installations or it is meant to be for individuals users? 
- What are migration plans for existing customers to support this phase out of Microsoft Store for Business?

**Reply by denelon** [reply]
WinGet has several interfaces for integration. I think the question is more on the configMrg (SCCM) side to see if they would like to integrate.
WinGet performs installations on the "local" machine but has APIs that can be called to perform those actions. It's the mechanism used by Intune for installing the new Store Apps.
The Intune team has been driving most of the efforts around the deprecation of the Microsoft Store for Business. As things get announced, I tend to add links here on relevant issues.

**Reply by techyrajeev** [reply]
We can have internal meeting if you want @denelon.

**Reply by denelon** [reply]
Sure, send me an e-mail. 😊
